webapp/maps_heatmaps/src/index.tsx

```diff
@@ -22,20 +22,8 @@
 // Imports
 let map: google.maps.Map, heatmap: google.maps.visualization.HeatmapLayer;
 import * as firebase from "firebase";
-import firebaseConfig from "./firebase_config";
-import * as geofirestore from "geofirestore";
```

Comment:
How come geofirestore is no longer imported (also not in declareDatabase.ts)?

Reply:
it is only used in queryDB so I removed It.